.github/workflows/continous-integration-docker.yml

```diff
@@ -575,3 +575,28 @@ jobs:
       root_state_path: '/var/cache'
       zebra_state_dir: 'zebrad-cache'
       lwd_state_dir: 'lwd-cache'
+
+  # Test that Zebra can handle a submit block RPC call, using a cached Zebra tip state
+  #
+  # Runs:
+  # - after every PR is merged to `main`
+  # - on every PR update
+  #
+  # If the state version has changed, waits for the new cached states to be created.
+  # Otherwise, if the state rebuild was skipped, runs immediately after the build job.
+  submit-block-test:
+    name: submit block test
+    needs: test-full-sync
+    uses: ./.github/workflows/deploy-gcp-tests.yml
+    if: ${{ !cancelled() && !failure() }}
```

Comment:
This job needs to be skipped when we're running manual test.
Can you change the job conditions to match similar jobs?

Reply:
Updated in [`d72ea40`](https://github.com/ZcashFoundation/zebra/pull/5589/commits/d72ea40650530d1dfa5e3bdd73d9d146eb28fba8). Looks like that's the case for every job in this file except `test-full-sync` and `regenerate-stateful-disks`. Noted.

Reply:
And `lightwalletd-transactions-test`, but we're going to revert that back to running on PRs.